### PR TITLE
Template for route short and long name equal

### PR DESCRIPTION
**Pull request by aannie-y**
Template for route short and long name having equal name.

This is what the updated template looks like:
image

### CODE REVIEW

File: webapp/src/main/webapp/resources/template.soy [hunk 1]
```diff
@@ -552,3 +552,36 @@
...
+  <p class="error">Error - Route Short and Long Name Equal!</p>
+  <p>Description: Both the short and long name of a route are equal.</p>
+  <p><b>{$numNotices}</b> found:</p>
+  <table>
+    <thead>
+      <tr>
+        <th>Route ID</th>
+        <th>CSV Row Number</th>
+        <th>Route Short Name</th>
+        <th>Route Long Name</th>
```
Comment (jupi3200) [inline webapp/src/main/webapp/resources/template.soy]: If they are equal, why do we need two columns?

Comment (aannie-y) [reply webapp/src/main/webapp/resources/template.soy]: True. I'll change it. Thanks

Comment (aannie-y) [reply webapp/src/main/webapp/resources/template.soy]: Done!

Comment (jupi3200) [reply webapp/src/main/webapp/resources/template.soy]: Can we update the screenshot?

**Review by jupi3200 — CHANGES_REQUESTED** [review]
Can you please update the screenshot?

**Comment by aannie-y** [comment]
> Can you please update the screenshot?

done

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**